stripe/src/main/java/com/stripe/android/view/ShippingFlowPagerAdapter.java

```diff
@@ -0,0 +1,93 @@
+package com.stripe.android.view;
+
+import android.content.Context;
+import android.support.annotation.Nullable;
+import android.support.v4.view.PagerAdapter;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.stripe.android.PaymentConfiguration;
+import com.stripe.android.R;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class ShippingFlowPagerAdapter extends PagerAdapter {
+
+    private Context mContext;
```

Comment:
Except in view / activity classes (where you have constructors forced on you without all the params), prefer to null-or-not annotate all the things